pages/en/lb4/Application.md

```diff
@@ -0,0 +1,211 @@
+---
+lang: en
+title: 'Application'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Application.html
+summary:
+---
+
+## What is an Application?
+
+In LoopBack 4, the [`Application`](http://apidocs.strongloop.com/@loopback%2fcore/#Application)
+class is the central class for setting up all of your module's components,
+controllers, servers and bindings. The `Application` class extends 
+[Context](Context.html), and provides the controls for starting and stopping
+itself and its associated servers.
+
+When using LoopBack 4, we strongly encourage you to create your own subclass
+of `Application` to better organize your configuration and setup.
+
+## Making your own application class
+
+By making your own application class, you can perform several additional
+tasks as a part of your setup:
+- Pass configuration into the base class constructor
+- Perform some asynchronous wireup before application start
+- Perform some graceful cleanup on application stop 
+
+{% include code-caption.html content="src/widget-application.ts" %}
+```ts
+import {Application} from '@loopback/core';
+import {RestComponent, RestServer} from '@loopback/rest';
+import {SamoflangeController, DoohickeyController} from './src/controllers';
+import {WidgetApi} from './apidef/';
+
+export class WidgetApplication extends Application {
+  constructor() {
+    // This is where you would pass configuration to the base constructor
+    // (as well as handle your own!)
+    super({
+      components: [RestComponent],
+    });
+    const app = this; // For clarity.
+    // You can bind to the Application-level context here.
+    // app.bind('foo').to(bar);
+    app.controller(SamoflangeController);
+    app.controller(DoohickeyController);
+  }
+
+  async start() {
+    // This is where you would asynchronously retrieve servers, providers and
+    // other components to configure them before launch.
+    const server = await app.getServer(RestServer);
+    server.bind('rest.port').to(8080);
+    server.api(WidgetApi);
+    return await super.start();
```

Comment:
Perhaps add a comment why is it important to call `super.start()`? (In order to let the base method to actually start the servers.)

Similar comment applies to `await super.stop()` below.